src/client/components/MealList.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+import "./styles.css";
+import MealCard from "./MealCard";
+
+function MealList() {
+  const [meals, setMeals] = useState([]);
+  useEffect(() => {
+    (async () => {
+      const res = await fetch("api/meals");
+      const data = await res.json();
```

Comment:
Why are you doing this instead of
```
useEffect( async () => {
    const res = await fetch("api/meals");
    const data = await res.json();
    setMeals(data.data);
}, [])
```

Reply:
thanks